src/processing/ObjectComparator.java

```diff
@@ -0,0 +1,25 @@
+package processing;
+
+import processing.utility.*;
+import java.util.*;
+
+/**
+* This is to compare two PriorityQueueObject objects, to store them in a sorted way according
+* to timestamp in the priority queue.
+*
+* @author Himanshu Jain
+*/
+
+public class ObjectComparator implements Comparator <PriorityQueueObject>{
+	
+	public int compare(PriorityQueueObject obj1, PriorityQueueObject obj2) {
```

Comment:
Can you put comments for the comparison, it would improve readability, else this file is great !